specs/openapi.yaml

```diff
@@ -0,0 +1,55 @@
+openapi: 3.0.1
+info:
+  title: K-mer Search API
+  version: 1.0.0
+servers:
+  - url: http://kmersearch-api-service/api/v1
+paths:
+  /search:
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SearchQuery'
+        required: true
+      responses:
```

Comment:
Sorry, I think we should also return 400 if the seq length is less than the term size etc.